lib/utils/encoding/sdkSuffix/revereseVersion.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @description A semVer like string, x.y.z or x.y is allowed
+ *              Reverses the version positions, x.y.z turns to z.y.x
+ * @param {string} versionString
+ * @return {string}
+ */
+function reverseVersion(versionString) {
+  let [major, minor, patch] = versionString.split('.');
+  let reversed = '';
+
+  if (patch) {
+    reversed = `${patch.padStart(2, '0')}${minor.padStart(2, '0')}${major.padStart(2, '0')}`;
+  } else {
+    reversed = `${minor.padStart(2, '0')}${major.padStart(2, '0')}`;
+  }
```

Comment:
May i suggest:
reversed = `${minor}${major}`
if (patch) reversed = `${patch}` + reversed

Just to avoid the minor/major duplicate code

Reply:
why not just:
```
versionString.split('.').reverse().forEach(function(part) {
    reversed += `${part.padStart(2, '0')}`;
}
```
?